endpoints/getting-started-grpc/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:alpine
+
+RUN apk update \
+  && apk add git
+
+COPY . /go/src/app
+
+# Don't do this in production! Use vendoring instead.
```

Comment:
The current policy is no vendoring for samples - not until we have a widely used vendoring solution. Reduces external dependencies to run sample code (or avoids having vendored code checked in).